frontend/src/pages/RoomPage/components/CollabContainer/components/PeerPrepCodeMirror.tsx

```diff
@@ -100,6 +101,28 @@ const PeerPrepCodeMirror = () => {
     }
   };
 
+  const submitCode = (src :string) => {
+    const mapping : {[key:string]: string} = {
```

Comment:
Can bring this mapping outside of the component, so that it won't create the mapping each time the function runs